kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ProcessServiceIntegrationTest.java

```diff
@@ -178,6 +189,28 @@ public void testAbortExistingProcess() {
             fail(e.getMessage());
         }
     }
+    
+    @Test()
+    public void testAbortExistingProcessWrongContainer() {
+        Map<String, Object> parameters = new HashMap<String, Object>();
+
+        Long processInstanceId = processClient.startProcess(CONTAINER_ID, PROCESS_ID_EVALUATION, parameters);
+        try {
+            assertNotNull(processInstanceId);
+            assertTrue(processInstanceId.longValue() > 0);
```

Comment:
unnecessary unboxing.
```suggestion
            assertTrue(processInstanceId > 0);
```